CMakeLists.txt

```diff
@@ -13,18 +13,18 @@
 cmake_minimum_required(VERSION 3.10)
 
 project(re
-  VERSION 2.8.0
+  VERSION 2.9.0
```

Comment:
What if the next version is 2.8.1 ?

Reply:
This would released from stable branch.

Reply:
So ist 2.9.0 correct?

Reply:
2.9.0-dev should be fine. so we can distinguish the stable release from dev versions